analyzers/tests/SonarAnalyzer.UnitTest/SymbolicExecution/Roslyn/RoslynSymbolicExecutionTest.Branching.LearnConstraint.cs

```diff
@@ -1241,6 +1241,26 @@ public void Branching_LearnsNumberConstraint_FromRanges_HasNumber(string express
         validator.ValidateTag("Arg", x => x.Should().HaveOnlyConstraints(ObjectConstraint.NotNull, NumberConstraint.From(expectedMin, expectedMax)));
     }
 
+    [TestMethod]
+    public void Branching_LearnsNumberConstraint_FromRanges_HasNumber1()
+    {
+        var code = """
+            if (i > 0)
+            {
+                if (i > 5)
```

Comment:
The code has changed behavior for `>`, `>=`, `<`, `<=`, but these UTs seems to cover only `>` case. There must be a bunch of cases that we don't assert.